ximage/tests/accessor/test_accessor_args.py

```diff
@@ -0,0 +1,71 @@
+import inspect
+
+from ximage.accessor.methods import XImage_Base_Accessor, XImage_DataArray_Accessor
+from ximage.labels.labels import label
+from ximage.patch.labels_patch import get_patches_from_labels
+from ximage.patch.labels_patch import get_patches_isel_dict_from_labels
+from ximage.labels.plot_labels import plot_labels
+
+
+# Utils functions ##############################################################
+
+
+def get_default_arguments_dict(function):
+    """Get default arguments of a function as a dictionary"""
+
+    signature = inspect.signature(function)
+    default_arguments = {}
+
+    for key, value in signature.parameters.items():
+        if value.default is not inspect.Parameter.empty:
+            default_arguments[key] = value.default
+
+    return default_arguments
+
+
+def compare_default_arguments(base_function, reference_function, changed_default_arguments={}):
+    """Check that default arguments of base_function and reference_function are the same"""
+
+    base_default_arguments = get_default_arguments_dict(base_function)
+    reference_default_arguments = get_default_arguments_dict(reference_function)
+
+    modified_default_arguments = {**base_default_arguments, **changed_default_arguments}
+
+    assert modified_default_arguments == reference_default_arguments
+
+
+# Tests for accessors' methods default arguments ###############################
+
+
+def test_base_accessor_label():
```

Comment:
Here we could do something more elaborate that we can also use in the GPM-API software.
For each class accessor, we retrieve the list of methods, loop over it, and call  `compare_default_arguments`.
So that if new methods are added, are automatically tested ;)

Reply:
@ghiggi Yes I can attempt something. I could use `inspect.getsourcelines` to retrieve the path and name of the imported method, which could be a bit dirty. Would you have another way of doing this in mind?

Reply:
No. I was thinking to the same approach you detailed. 
And also let's do a test where we list the currently implemented methods, and we check that they exists in the accessor, so that we can better track future changes in methods' names.

Reply:
The thing is that for now, I am not aware of a way to find out that, for example, the `label_patches` accessor's method must call `get_patches_from_labels` (a different method name) from `ximage.patch.labels_patch` (an arbitrary path), without needing to read the source code of the accessor's method.

Also, would you see an issue in rewriting the accessor's method definition as something along the lines of
```
from ximage.patch.labels_patch import get_patches_from_labels
self.label_patches = functools.partial(get_patches_from_labels, self._obj)
```
? I am not 100% sure it works, but it would remove the need from having to copy the list of arguments.

Reply:
Mmh we can try out. 
I propose the following @sphamba:
- we merge the current PR 
- we address the testing of the accessors in a separate PR (where I can also experiment a couple of stuff ...)

Do you agree?
We could explore doing the method assignment in the `__init__` maybe, and also copy the docstring to the method!

Reply:
Sounds good! Ready to merge